index.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module 'nebenan-redux-tools/lib/network' {
```

Comment:
https://github.com/good-hood-gmbh/nebenan-frontend/blob/fbe013e0d506f7a29ba0f9de965e3a1a52b3209f/index.d.ts#L18

Reply:
Didn't quiet got this one

Reply:
just copy paste it from coreFE. then you don't need to type it yourself - also the type from CoreFE is way more complete

Reply:
Have decided to leave it the way it is right now, since from the core it requires to have "axios" installed for the proper types, which we are not using in nebenan-services. 
Will keep it the way it is right now, and will properly update when will typescriptify `nebenan-redux-tools`